src/Microsoft.Data.Analysis/PrimitiveDataFrameColumnArithmetic.tt

```diff
@@ -45,6 +45,10 @@ namespace Microsoft.Data.Analysis
                 return (IPrimitiveDataFrameColumnArithmetic<T>)new <#=type.ClassPrefix#>Arithmetic();
             }
 <# } #>
+            else if (typeof(T) == typeof(DateTime))
```

Comment:
We shouldn't need that since `DateTime` type is added to ColumnArithmeticTemplate already

Reply:
Thanks!